test/fixtures/approvals/v3_exercise_does_not_exist.approved.json

```diff
@@ -1,7 +1,7 @@
 {
   "exercise": {
     "track_id": "fake",
-    "language": "Fake",
+    "language": "Unknown",
```

Comment:
In this case the language is incorrect that is why is Unknown, I'll will add a test for that case as well